src/sap.ui.core/src/sap/ui/core/Popup.js

```diff
@@ -3018,24 +3018,24 @@ sap.ui.define([
 	 * with buttons it has to be checked whether a button or content-element
 	 * is available that can be focused.
 	 *
-	 * @param [object]
+	 * @param {object}
 	 *            mParameters contain all necessary parameters
-	 * @param [object.object]
+	 * @param {object.object}
```

Comment:
The object.object notation is also invalid in JSDoc. The type should only be the type of the param itself.

Reply:
Yeah i noticed this ALOT in the jsdocs - is there an official way or statement how they should be implemented? Got any example iimplementation where I could copy the style requested for this?

Reply:
Either see the explanation of @type at https://jsdoc.app or the samples in our control development guidelines at https://github.com/SAP/openui5/blob/master/docs/guidelines/classexample.md. 

I already created an internal change from your PR and pushed it for internal review. While doing so, I fixed the {object.object} etc. types. I just wondered if you want to push more changes, then I'll wait with submitting this one.

Reply:
it depends - if you already fixed the types definition then you can go ahead and internally send it for review - for other modules or occourences I'll try to implement them as suggested in the guidelines you posted above and open a separate PR for that.

...except you want a collective PR for all jsdoc changes?

Reply:
I guess I'll go ahead wit this PR and you might collect further fixes in a new PR - if that's okay with you?

Reply:
sure